modules/misunderstood/src/views/full/MainScreen/style.scss

```diff
@@ -48,9 +48,16 @@
 }
 
 .chatPreviewText {
+  width: 100%;
   padding: 5px;
-  border-radius: 1px;
+  border-radius: 15px;
+  text-align: center;
+  padding: 8px;
   border: 1px solid #ccc;
+
+  p {
+    margin: 0;
```

Comment:
Somehow bootstrap was adding a margin to the p element that was causing the text inside each bubbles to be wrongly aligned.